src/esn.calendar.libs/app/components/event/form/event-form.controller.js

```diff
@@ -668,4 +677,38 @@ function CalEventFormController(
       placement: 'center'
     });
   }
+
+  /**
+   * Checks if an event can be moved into another calendar
+   *
+   * @param {boolean} success - true if the previous response was successful
+   *
+   * @returns {Promise}
+   */
+  function canPerformCalendarMove(success) {
+    if (!success) return $q.when(false);
+
+    return _calendarHasChanged() ? changeCalendar() : Promise.resolve();
```

Comment:
Is there a specific reason to use `Promise.resolve` and `$q.when` on the same function? They both do almost the same thing.

Reply:
no reason, i didn't pay attention.